src/components/profile/ProfileService.js

```diff
@@ -256,11 +269,16 @@ goog.require('ga_urlutils_service');
         };
 
         this.get = function(feature) {
+          if (!feature || !feature.getGeometry()) {
+            var defer = $q.defer();
+            defer.resolve(emptyData);
+            return defer.promise;
+          }
           var coordinates = feature.getGeometry().getCoordinates();
 
           // TODO: manage all kind of geometry
-          if (feature.getGeometry() instanceof ol.geom.Polygon ||
-              feature.getGeometry() instanceof ol.geom.LinearRing) {
+          if (feature && (feature.getGeometry() instanceof ol.geom.Polygon ||
```

Comment:
check for feature not needed as you added ` if (!feature || !feature.getGeometry())` just above